pkg/config/controller_config.go

```diff
@@ -23,6 +23,7 @@ const (
 	CLUSTER_VPC_ID                  = "CLUSTER_VPC_ID"
 	CLUSTER_NAME                    = "CLUSTER_NAME"
 	DEFAULT_SERVICE_NETWORK         = "DEFAULT_SERVICE_NETWORK"
+	SECONDARY_ACCOUNT_TEST_ROLE_ARN = "SECONDARY_ACCOUNT_TEST_ROLE_ARN"
```

Comment:
should be in test package not source code

Reply:
Will put this in Framework